templates/loginform.php

```diff
@@ -201,7 +201,7 @@ class="float-l erroricon" alt="gtk-dialog-error"/>
                                     echo htmlspecialchars($this->t('{privacyidea:privacyidea:scanTokenQR}'));
                                     ?>
                                     <div class="tokenQR">
-                                        <?php echo '<img src="' . $this->data['tokenQR'] . '" />'; ?>
+                                        <?php echo '<img php-client="' . $this->data['tokenQR'] . '" />'; ?>
```

Comment:
i think this is an rename error?

Reply:
This src is html tag's attribute.

Example:
 <img src="img_girl.jpg" alt="Girl in a jacket" width="500" height="600">

Reply:
` <img src="img_girl.jpg" alt="Girl in a jacket" width="500" height="600"> `